interfaces/interface_ids.ts

```diff
@@ -0,0 +1,10 @@
+export const LSP_INTERFACE_IDS = {
```

Comment:
Why not using the interface IDs of the lsp-smart-contracts package? Might be easier to maintain.

Reply:
Yes this is a good point indeed